config/config.go

```diff
@@ -33,6 +33,9 @@ import (
 // Devnet identifies the 'development network' use for development and not generally accessible publicly
 const Devnet protocol.NetworkID = "devnet"
 
+// Betanet identifies the 'beta network' use for early releases of feature to the public prior to releasing these to mainnet/testnet
+const Betanet protocol.NetworkID = "betanet"
+
 // Devtestnet identifies the 'development network for tests' use for running tests against development and not generally accessible publicly
 const Devtestnet protocol.NetworkID = "devtestnet"
```

Comment:
Is `devtestnet` obsolete?  should it be removed?

Reply:
No, as it's being used by multiple unit tests.